osf-cli/pom.xml

```diff
@@ -0,0 +1,120 @@
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>osf-cli</artifactId>
+    <packaging>jar</packaging>
+    <name>OSF Package Tool Command Line</name>
+
+    <parent>
+        <groupId>org.dataconservancy.cos</groupId>
+        <artifactId>osf-packaging</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <!-- Properties ================================================== -->
+
+
+    <properties>
+        <!-- definition of assembly related properties -->
+        <pkg.gen.assembly.standalone.id>standalone</pkg.gen.assembly.standalone.id>
+        <pkg.gen.assembly.standalone.fileName>${project.artifactId}</pkg.gen.assembly.standalone.fileName>
+
+    </properties>
+    <!-- Profiles  =================================================== -->
+
+    <!-- Build ======================================================= -->
+
+    <build>
+        <plugins>
+
+            <plugin>
+                <!-- Create the property $buildNumber holding the current Git revision -->
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <doCheck>false</doCheck>
+                    <doUpdate>false</doUpdate>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
+                    <appendAssemblyId>false</appendAssemblyId>
+                    <attach>true</attach>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>standalone-jar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <archive>
+                                <manifest>
+                                    <mainClass>org.dataconservancy.cos.packaging.cli.PackageGenerationApp</mainClass>
+                                    <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                                    <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                                </manifest>
+                                <manifestEntries>
+                                    <Implementation-Title>${project.name}</Implementation-Title>
+                                    <Implementation-Version>${project.version}
```

Comment:
This produces a MANIFEST.MF that looks like:

```
Manifest-Version: 1.0
Implementation-Title: OSF Package Tool Command Line
Implementation-Version: 1.0.0-SNAPSHOT
                               
          84c0e36e01a030721ab94d7b1e33e2b7222dcb4e
Archiver-Version: Plexus Archiver
Built-By: esm
Specification-Vendor: Data Conservancy
Specification-Title: OSF Package Tool Command Line
Implementation-Vendor-Id: org.dataconservancy.cos
Implementation-Vendor: Data Conservancy
Main-Class: org.dataconservancy.cos.packaging.cli.PackageGenerationApp
Created-By: Apache Maven 3.3.9
Build-Jdk: 1.8.0_73
Specification-Version: 1.0.0-SNAPSHOT
```

With weird spacing around `Implementation-Version:`, maybe something like:

    Implementation-Version: 1.0.0-SNAPSHOT (<commit hash here>)